### ISSUE TITLE
Lien "Voir dans le navigateur" erroné sur les emails des indicateurs

### ISSUE THREAD

**Issue by fcoufour** [betagouv/recosante-mail#83]
Le lien redirige vers une 404 (https://recosante.beta.gouv.fr/places/50218/Granville) au lieu de rediriger vers le "dashboard" de la commune (https://recosante.beta.gouv.fr/place/50218/granville/).

`place` au lieu de `places` et `slug` de la commune tout en minuscule.

**Comment by florianpanchout** [comment]
Modifié dans le repo (https://github.com/betagouv/recosante-mail/blob/master/src/pages/indicateurs.html), mais impossible de l'éditer sur Sendinblue. L'éditeur texte n'est plus le même et ne répond pas correctement quand j'essaie de faire une modification. Tu as changé quelque chose @fcoufour ?

**Comment by fcoufour** [comment]
J'ai seulement fait une copie du template (1794) pour tester mes modifications sans impacter le reste. J'en encore l'éditeur "classic" pour la copie.

**Comment by fcoufour** [comment]
Visiblement y'a un peu plus de traitement que le `lower` :

```
${place.nom
              .toLowerCase()
              .replace(/\s/g, '-')
              .replace(/'/g, '-')
              .normalize('NFD')
              .replace(/[\u0300-\u036f]/g, '')}/`
```

**Comment by fcoufour** [comment]
Dernière modification le "23 Déc 2021, 10:31" sur le template 1465 donc probablement SIB

**Comment by florianpanchout** [comment]
Effectivement, j'étais un peu ambitieux avec le lower... Du coup il me faut un nouvel attribut user avec l'url du dashboard associé (je ne peux pas faire ces opérations dans un template de mail)

**Comment by florianpanchout** [comment]
En fait ces deux lignes remplacent tous les caractères spéciaux, accents compris. Donc oui, il y a des centaines de communes qui en ont besoin pour leur url

**Comment by fcoufour** [comment]
Indeed, on va donc faire l'équivalent Python

**Comment by fcoufour** [comment]
Voir côté API : https://github.com/betagouv/recosante-api/pull/334
Voir côté front-end : https://github.com/betagouv/recosante/pull/185
